recipes/omniorbpy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "4.2.3" %}
+
+package:
+  name: omniorbpy
+  version: {{ version }}
+
+source:
+#  fn: omniORB-{{ version }}-{{ release }}.tar.gz
+  url: https://sourceforge.net/projects/omniorb/files/omniORBpy/omniORBpy-{{ version }}/omniORBpy-{{ version }}.tar.bz2
+  sha256: 5c601888e57c7664324357a1be50f2739c468057b46fba29821a25069fc0aee5
+  patches:
+    - patch-configure.diff
+    - patch-dir.mk
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - omniorb
```

Comment:
`omniorb` is probably a `host` requirement

Reply:
thanks! trying as we speak...

Reply:
@chrisburr what if I want to build omniorbpy locally? Does it suffice to say:

```
conda build -c conda-forge -c defaults recipes/omniorbpy --python=3.6
```

or do I have to instruct conda-build to pick omniorb locally (I have already built it with a similar command)?

Reply:
It is possible to to get conda-build to automatically build dependencies but I'm not I don't know exactly what heuristics it uses. (It might require you to do `cd recipes` first.)

This should definitely work:
```bash
conda build -c conda-forge -c defaults recipes/omniorb --python=3.6
conda build -c local -c conda-forge -c defaults recipes/omniorbpy --python=3.6
```
(I think local is added by default for `conda build` but not for `conda install`.)